docs/console-api/openapi/experiments/experiments-data.json

```diff
@@ -2107,7 +2107,7 @@
               }
             }
           },
-          "description": "This endpoint will add overrides to the existing list of overrides."
+          "description": "This endpoint will add overrides to the existing list of overrides. If there are overrides already exists, they will be replaced with new ones."
```

Comment:
I think the first sentence seems wrong - maybe something more like just "This will overwrite the existing overrides with new ones."

Reply:
since it can be a list of ids and entities, the ones that are not exist will be added and the ones existed (same id, type, env...) will be replaced with new ones. So I think it's still valid. The most confused part to me is PATCH. With this behavior it should be PUT or POST

Reply:
i think we can just say that any conflicting overrides are removed, but i dont know if we really need to clarify that